src/main/java/org/team1540/elmo/ejectors/EjectInner.java

```diff
@@ -0,0 +1,22 @@
+package org.team1540.elmo.ejectors;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class EjectInner extends SequentialCommandGroup {
+
+    BunnyEjectors.InnerEjector innerEjector;
+
+    public EjectInner(BunnyEjectors.InnerEjector innerEjector) {
+        this.innerEjector = innerEjector;
+        addRequirements(innerEjector);
+
+        addCommands(
+                new InstantCommand(()->innerEjector.setSuction(false)),
```

Comment:
Wouldn't it want to extend the arm first and then release the suction to get to the inner scoring area?

Reply:
For this one, the suction cup is fixed to the robot and the piston pushes the tube off after its unsuctioned

Reply:
But shouldn't the class titled "EjectInner" be ejecting into the inner scoring area, which would be using the arm that extends with suction

Reply:
nevermind i mixed up hte names